sys/mips/mips/pmap.c

```diff
@@ -2618,13 +2618,7 @@ pmap_copy_page_internal(vm_page_t src, vm_page_t dst, int flags)
 		    MIPS_PHYS_TO_DIRECT(phys_dst), PAGE_SIZE);
 		va_src = MIPS_PHYS_TO_DIRECT(phys_src);
 		va_dst = MIPS_PHYS_TO_DIRECT(phys_dst);
-#ifdef CPU_CHERI
-		if (flags & PMAP_COPY_TAGS)
```

Comment:
Do we always need to preserve tags here?

Reply:
No, indeed this was a mistake.

Reply:
It looks like that stripping tags here leads to init crashing. Will investigate this.